### PR TITLE
Rename and remove blob APIs

### PR DESCRIPTION
**Pull request by sakirsch-msft**
Rename already existing blob APIs to new style and remove container/blob Get Metadata APIs.

Containers_ListBlobs will eventually be split into two different APIs, but in the meantime it has been renamed to one of the two.

### CODE REVIEW

File: Specs/blob/blob-storage-2017-07-29.json [hunk 1]
```diff
@@ -983,8 +925,8 @@
         "tags": [
           "containers"
         ],
-        "operationId": "Containers_ListBlobs",
-        "description": "The List Blobs operation returns a list of the blobs under the specified container",
+        "operationId": "Containers_ListBlobFlatSegment",
```
Comment (PaulWells) [inline Specs/blob/blob-storage-2017-07-29.json]: There should be a Containers_ListBlobHierarchySegment as well. Can we add that as part of this change?  It looks like Containers_ListBlobFlatSegment should not have the delimiter parameter and Containers_ListBlobHierarchySegment should have the delimiter parameter. And then some of the other parameters / request body should only be in one or the other.

Comment (sakirsch-msft) [reply Specs/blob/blob-storage-2017-07-29.json]: Yep, I had mentioned that in the description as I was planning to add it in the PR that adds new operations.

Happy to add it here instead.

Comment (PaulWells) [reply Specs/blob/blob-storage-2017-07-29.json]: Or we can add a task to this PBI for doing that. I have no preference

Comment (PaulWells) [reply Specs/blob/blob-storage-2017-07-29.json]: Okay, in that case nevermind

Comment (sakirsch-msft) [reply Specs/blob/blob-storage-2017-07-29.json]: As per face-to-face we will take this in a future PR that adds new APIs.

**Pull request merged**